### PR TITLE
Commands: guard against constructor being the target in those that don't

### PR DESCRIPTION
**Pull request by Morfent**
...even though the results are pretty entertaining for a couple of them.

### CODE REVIEW

**Comment by Zarel** [comment]
This is the wrong level to be doing this ate

**Comment by Morfent** [comment]
What do you mean?

**Comment by Zarel** [comment]
I mean, fix `Tools.get...()`

**Comment by Zarel** [comment]
I'm already doing that, anyway, so it might be best to just wait

**Comment by Morfent** [comment]
Removed the changes that wouldn't be needed when `Tools.get` gets fixed.

File: commands.js [hunk 1]
```diff
@@ -1207,6 +1208,12 @@ exports.commands = {
 		let targetUsername = (targetUser ? targetUser.name : target);
 
 		let buffer = [];
+		if (targetId === 'constructor') {
+			buffer.push('constructor user auth:');
+			buffer.push('No global or room auth.');
+			return connection.popup(buffer.join('\n\n'));
+		}
+
```
Comment (Zarel) [inline commands.js]: This is a lot of hacking...

Comment (panpawn) [inline commands.js]: Why not just ``return false;`` or something?

Comment (bumbadadabum) [inline commands.js]: If you're going to do this much hacking you may as well let it return "nice
try" or something

**Comment by Morfent** [comment]
This should be fine now

**Pull request merged**